modules/hostap/src/supp_main.c

```diff
@@ -43,8 +43,8 @@ K_SEM_DEFINE(z_wpas_ready_sem, 0, 1);
 #include <l2_packet/l2_packet.h>
 
 /* Should match with the driver name */
-#define DEFAULT_IFACE_NAME "wlan0"
-#define IFACE_MATCHING_PREFIX "wlan"
+#define DEFAULT_IFACE_NAME "nordic_wlan0"
```

Comment:
How does the DTS entry map to iface name? I don't see anywhere we are setting the iface name actively?

Reply:
Created an entry in chosen node for `zephyr,wifi` which can be referred for fetching the device info.